main.go

```diff
@@ -405,6 +405,103 @@ func init() {
 	})
 
 	rootCmd.AddCommand(volumeCmd)
+
+	endpointCommand := &cobra.Command{
+		Use:     "endpoint",
+		Short:   "Manage endpoints in Unweave",
+		GroupID: groupDev,
+		Aliases: []string{"end"},
+		Args:    cobra.NoArgs,
+	}
+
+	endpointNewCmd := &cobra.Command{
+		Use:   "new <exec-id>",
+		Short: "Create a new endpoint in Unweave",
+		Long: wordwrap.String("Create a new endpoint in Unweave.\n\n"+
+			"Eg. unweave endpoint new <exec-id>\n\n",
+			ui.MaxOutputLineLength),
+		Args:    cobra.ExactArgs(1),
+		Aliases: []string{"new", "n", "create", "c"},
+		RunE:    cmd.EndpointCreate,
+	}
```

Comment:
This is a wip syntax right? I think the way we want to expose this is probably: 
`unweave endpoint new <endpoint-name>` and then `unweave deploy <endpoint-name>`.

Reply:
Yes, likely. Once you've got the ERD for this we can hash out the details, and I can make the changes easily :100: